source/styleEditor.js

```diff
@@ -1,3 +1,11 @@
-$(document).ready(function(){
-  // your code goes here.
-});
+$(document).ready(function() {
+    let selector = $("input[name='selector']")
+    let property = $("input[name='property']")
+    let value = $("input[name='value']")
+    let submit = $("input[type='submit']")
```

Comment:
I like waiting to use `val()`, would be more flexible in the future if those are needed again.